Umbraco-Cloud/Set-Up/Manage-Hostnames/index.md

```diff
@@ -41,9 +41,13 @@ You will also have to specify the hostname for each root node if you are using a
 
 Once you've assigned a hostname to your Umbraco Cloud environment, you may want to hide the default umbraco.io URL (e.g. *snoopy.s1.umbraco.io*). We've created a rewrite rule for this purpose - find it in the [Rewrites on Cloud](Rewrites-on-Cloud/#hiding-the-default-umbraco-io-url) article.
 
-## [Umbraco Latch](../Umbraco-Latch)
+### Automatic TLS (HTTPS)
+
+All hostnames added to an Umbraco Cloud project's environment will get a TLS (HTTPS) certificate added by default. The certificate is issued by Cloudflare and valid for 1 year after which it will be automatically renewed. Everything around certificates and renewals is handled for you, so the only thing you need to worry about is updating the DNS records for your hostname.
```

Comment:
```suggestion
All hostnames added to a Umbraco Cloud project's environment will get a TLS (HTTPS) certificate added by default. The certificate is issued by Cloudflare and valid for 1 year after which it will be automatically renewed. Everything around certificates and renewals is handled for you and you only need to make sure that the DNS records are configured according to our recommendations listed above.
```